### DISCUSSION TITLE
Projects within projects

### DISCUSSION THREAD

**Discussion by jvvk** [quarto-dev/quarto-cli#788] [Q&A] [answered]
Can I have a book project within a website project?
What is the best way to manage multiple quatro book projects within a quatro website project?

**Comment by jjallaire** [comment]
I would suggest creating an explicit render list for the quarto website that _excludes_ the book directories. For example:

```
project:
    render:
          - *.qmd
          - !books/
```

**Reply by jvvk** [reply]
Thank you very much!I am trying to use book projects to simulate the concept of series in Alison Hill’s Hugo Apero theme. 
For deployment to Netlify, I would then have to redirect the render output of each of the book projects to the website project’s _site directory right?

**Comment by cderv** [comment, marked as answer]
FWIW the [rstudio.github.io/r-manuals](https://rstudio.github.io/r-manuals) (https://github.com/rstudio/r-manuals) example from the gallery is a website that organize a collection of books. You can look at the R script that organize the workflow. We first generate the books using a template YAML and then include the HTML books into the website before rendering the website. 

There is probably other ways to do this meta organization but I figured this one is worth sharing as it worked fine for us.

**Reply by jvvk** [reply]
Thank you very much!
